### ISSUE TITLE
Huge Scaling difference for the 100 dist histogram version 

### ISSUE THREAD

**Issue by benfig1127** [benfig1127/Summer-Research-2020#34]
Hey Mike, I bumped the k off diag distances to 100, which seems to not move the median a bit, however unless we store probably over 1000+ k off diag distances I dont think I can get the median to 2.0. The other slight problem is there is now a massive difference in the size of the diag distances and the off diag distances. Can you point me towards how you want this fixed? 

![Figure_1-1](https://user-images.githubusercontent.com/61296503/92990278-e3f1d980-f48f-11ea-8868-e0bfa48ad96b.png)

**Comment by mikeizbicki** [comment]
Change the x-axis so that it starts at 0 (but leave the end-value as-is), and adjust the y-axis scaling so that the peaks of both histograms are normalized to one.  Remove the label and y-tics from the  y-axis.  Also fix your spelling in the key.

**Comment by benfig1127** [comment]
I am not sure what you mean by normalized to one, I can use the density paramater in matplotlib, which is given here: 

"If True, the first element of the return tuple will be the counts normalized to form a probability density, i.e., the area (or integral) under the histogram will sum to 1. This is achieved by dividing the count by the number of observations times the bin width and not dividing by the total number of observations. If stacked is also True, the sum of the histograms is normalized to 1.

Default is None for both normed and density. If either is set, then that value will be used. If neither are set, then the args will be treated as False.

If both density and normed are set an error is raised."


Which outputs this 
![Figure_1-2](https://user-images.githubusercontent.com/61296503/92990543-3207dc80-f492-11ea-86a4-778d4669e43f.png)

but I am not sure this is what you want

**Comment by benfig1127** [comment]
Also part of the submission process asks for "Type (or cut-and-paste) a short text summary for your submission. It should not include the author/title information, which was already entered above." 
Should this just be our abstract or something else?

**Comment by mikeizbicki** [comment]
That plot is fine if you fix everything else and adjust the x-axis range.

Yes, that's the abstract.

**Comment by benfig1127** [comment]
What should the y axis be called then

**Comment by benfig1127** [comment]
Is this one fine?
![Figure_1-3](https://user-images.githubusercontent.com/61296503/92990828-5a90d600-f494-11ea-80de-cf4a9bf7ea22.png)

**Comment by benfig1127** [comment]
So I was adding the things to the submission link that I could ie authors etc. And I thought I could save it before the final submission. I haven't actually attached any files, but this message is slightly scary. Im assuming its perfectly fine to have my submission be blank up until the deadline and we can simply revise the submission with the actual final paper then (ie as long as our paper is attached before the final deadline we wont have any problems) 
![Screenshot from 2020-09-12 01-15-24](https://user-images.githubusercontent.com/61296503/92991078-e7885f00-f495-11ea-9079-659b2476f24b.png)

**Comment by mikeizbicki** [comment]
No, the y-axis should have no label and no x-tics.  The x-axis should be redone to show the whole orange distribution (max 1.3?)

Yes, you can update the submission as many times as you'd like.

**Comment by benfig1127** [comment]
Okay thanks, sorry I freaked out for a second on the submission thing, so like this instead?
![Figure_1-4](https://user-images.githubusercontent.com/61296503/92991439-a04f9d80-f498-11ea-860e-58816a70191a.png)

**Comment by mikeizbicki** [comment]
And now make the font sizes reasonable.

**Comment by benfig1127** [comment]
![Figure_1-5](https://user-images.githubusercontent.com/61296503/92992171-d9d6d780-f49d-11ea-9c32-75afb132e3be.png)

**Comment by benfig1127** [comment]
Overleaf tex file has been updated with final version of the histogram plot for you. I will be periodically adding the accuracies over the next 24/36 hours and will add in the FIXME numbers before the AM. To clarify, the deadline is the Tuesday the 15th, so I should have plenty of time to wait for the final accuracies to flow in. I will leave any comments on the paper's written contents I have in another issue tomorrow.
